### PR TITLE
Updates for start of 2023-2034

### PR DESCRIPTION
**Pull request by Rmalavathu**
Updates Officers Page for 2023-2034
Remove Part Page and added Neurotech
Updates ROV page
Added callouts for ROV and Racing

### CODE REVIEW

File: assets/calendar.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ const all_cal = [
     { name: 'mtt-s', color: '0D7813', src: 'edv88gqt4b1v7fs6b6gi2bd8f8@group.calendar.google.com'},
     // { name: 'part', color: 'A32929', src: 'h8nk778rgq4idlspuf4uevfs84@group.calendar.google.com'},
```
Comment (hadiahmed098) [inline assets/calendar.js]: Just remove PART, don't comment it out

File: assets/include.js [hunk 2]
```diff
@@ -35,7 +35,7 @@ Vue.component('ieee-header', {
                     <li><a class="dropdown-item" href="/growth">Growth & Engagement</a></li>
                     <li><hr class="dropdown-divider"></li>
                     <li><div class="dropdown-header py-1">Technical</div></li>
-                    <li><a class="dropdown-item" href="/part">Aerial Robotics</a></li>
+                    <li><a class="dropdown-item" href="/Neurotech">Neurotech Committee</a></li>
```
Comment (hadiahmed098) [inline assets/include.js]: No capitals in URLs

Comment (hadiahmed098) [inline assets/include.js]: Don't say "Neurotech Committee" just go with "Neurotech". The committee part is implied with the heading

File: neurotech/index.html [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+                        </div>
+                        <div class="col-4">
+                            <img src="./images/carousel/3.jpg" alt="Neurotech Photo 3" width="400">
+                        </div>
+                    </div>
+                    <div class="row justify-content-center align-items-center g-2">
+                        <div class="col-4">
+                            <img src="./images/carousel/4.jpg" alt="Neurotech Photo 4" width="400">
+                        </div>
+                    </div>
```
Comment (hadiahmed098) [inline neurotech/index.html:53]: Not sure what the purpose of this `div` is, just combine it with the row from above

File: neurotech/index.html [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+                </div>
+                <p class="display-6 text-light">Reach out to <a href="mailto:asultanb@purdue.edu">asultanb@purdue.edu</a> if interested.</p>
```
Comment (hadiahmed098) [inline neurotech/index.html]: Designate a committee alias for this, not a personal email.

File: neurotech/index.html [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+                <p class="fs-4 text-light">Return to the <a href="/">home page</a>.</p>
```
Comment (hadiahmed098) [inline neurotech/index.html]: This link to the home page is completely unnecessary and not standard to the other pages.

File: neurotech/index.html [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+            </div>
+        </main>
```
Comment (hadiahmed098) [inline neurotech/index.html:69]: Missing a section at the bottom for leadership

image

File: neurotech/index.html [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+        <script src="/assets/external/vue2.min.js"></script>
+        <script src="/assets/include.js"></script>
+
+    </head>
+    <body>
+    <div id="app" class="bg-dark">
+        <ieee-header head="Purdue IEEE" page="Neurotech"></ieee-header>
+
+        <main>
+            <div class="container-fluid pt-5 mt-5 text-center bg-dark">
```
Comment (hadiahmed098) [inline neurotech/index.html]: This section doesn't follow the existing format of the other tech committee pages.

There should be the logo + title in the top bar with nav links, then a paragraph of text describing the committee with a catchy marketing title.

File: neurotech/neurotech.css
Comment (hadiahmed098) [inline neurotech/neurotech.css:1]: Empty custom CSS file. Avoid having custom CSS for each committee. Defer to the master style sheet.

File: officers/2022.json [hunk 8]
```diff
@@ -48,7 +48,14 @@
     "committees": "<a href=\"/part/\">Aerial Robotics</a>",
     "email": "sec@purdueieee.org"
   }],
-  "cornerstone":[
+  "cornerstone":[{
+    "name": "Rohan Malavathu",
+    "image": "/images/officers/2022_Rohan_Malavathu.jpg",
+    "position": "Infrastructure Lead",
+    "description": "Rohan is a sophomore in computer engineering. He has an interest in software development and machine learning. In his free time he likes to cook and watch sports.",
+    "committees": "<a href=\"/part/\">Aerial Robotics</a>",
```
Comment (hadiahmed098) [inline officers/2022.json]: Update all links to PART with a link to the PART's own website.

File: officers/2022.json [hunk 9]
```diff
@@ -48,7 +48,14 @@
...
+    "email": "infrastructure@purdueieee.org"
```
Comment (hadiahmed098) [inline officers/2022.json]: Not the alias? Unless you are making a new one.

File: rov/index.html [hunk 10]
```diff
@@ -70,14 +70,14 @@ <h2 class="fs-1">Creating robots for the most extreme environments.</h2>
                             </p>
                         </div>
                         <div class="col-md-5">
-                            <img class="rounded img-fluid" src="./images/2021_Team_Competition.jpg" alt="2021 Competition Team">
```
Comment (hadiahmed098) [inline rov/index.html:73]: You may want to purge old photos to avoid having a bunch in the git repo

File: sitemap.xml [hunk 11]
```diff
@@ -41,7 +41,7 @@
 	<priority>0.8</priority>
 </url>
 <url>
-	<loc>https://purdueieee.org/part/</loc>
+	<loc>https://purdueieee.org/neurotech/</loc>
```
Comment (hadiahmed098) [inline sitemap.xml:44]: Don't modify the sitemap like this, [use a site like this](https://www.xml-sitemaps.com/) instead

**Review by hadiahmed098 — CHANGES_REQUESTED** [review]
General feedback:
1. You need to update the leadership at the bottom of each page to match the current officer list
2. Either in this PR or one soon after, update all links to `/part` with one to `https://purdueaerial.com`, similar to how was done for Orbital after they split

File: neurotech/index.html (hunk 7 above)
Comment (adithyachan) [reply neurotech/index.html]: We currently do not have a slogan for Neurotech and are planning on moving away from the top left nav bar design. For the time being, Neurotech will be an outlier however, we plan to revamp the style of all the other pages to maintain a uniform updated style within the next few weeks.

File: assets/calendar.js (hunk 1 above)
Comment (adithyachan) [reply assets/calendar.js]: Resolved

File: assets/include.js (hunk 2 above)
Comment (adithyachan) [reply assets/include.js]: Resolved

Comment (adithyachan) [reply assets/include.js]: resolved

File: neurotech/index.html (hunk 5 above)
Comment (adithyachan) [reply neurotech/index.html]: resolved

File: neurotech/index.html (hunk 4 above)
Comment (adithyachan) [reply neurotech/index.html]: resolved

File: neurotech/index.html (hunk 3 above)
Comment (adithyachan) [reply neurotech/index.html:53]: resolved

File: neurotech/neurotech.css
Comment (adithyachan) [reply neurotech/neurotech.css:1]: Needed this for a scroll animation I wanted to implement

File: officers/2022.json (hunk 8 above)
Comment (adithyachan) [reply officers/2022.json]: Resolved

File: officers/2022.json (hunk 9 above)
Comment (adithyachan) [reply officers/2022.json]: resolved

File: rov/index.html (hunk 10 above)
Comment (adithyachan) [reply rov/index.html:73]: problem for later

File: sitemap.xml (hunk 11 above)
Comment (adithyachan) [reply sitemap.xml:44]: We'll do this after we get this PR pushed

**Review by hadiahmed098 — APPROVED** [review]
Looks good, hope the new website looks even better!

File: neurotech/index.html (hunk 6 above)
Comment (Rmalavathu) [reply neurotech/index.html:69]: Yeah I haven't heard anything from the chair and I am going to leave them off until they give me the proper information

File: neurotech/index.html (hunk 7 above)
Comment (Rmalavathu) [reply neurotech/index.html]: Yeah I think it is fine for now it is similar to what we did to software saturdays

**Comment by adithyachan** [comment]
Thanks Hadi! Also, as always, appreciate the thorough code review.

**Pull request merged**